tox.ini

```diff
@@ -9,7 +9,7 @@ deps =
     pylint: pylint>=1.4,<1.5
     py26: unittest2
 commands =
-    test: python -c "import h5py; h5py.run_tests()"
+    test: python -c "from sys import exit; import h5py; exit(0) if h5py.run_tests().wasSuccessful() else exit(1)"
```

Comment:
I think this `exit(0) if h5py.run_tests().wasSuccessful() else exit(1)`
could be simplified to `exit(not h5py.run_tests().wasSuccessful())`.
Probably could use a relative import to make this shorter still.